libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -48,7 +47,7 @@ class AC_PosControl
 public:
 
     /// Constructor
-    AC_PosControl(const AP_AHRS_View& ahrs, const AP_InertialNav& inav,
+    AC_PosControl(AP_AHRS_NavEKF& ahrs, const AP_AHRS_View& ahrs_view,
```

Comment:
Yes, I understand, it is just that ahrs_view is mostly a copying what is in ahrs, so if we could have remove ahrs_view, we could have simplied the codebase a little but that is not possible at least at one place the rotation from ahrs_view is used ! So we could do the other way and only keep ahrs_view ! But we will need to carefully review the constructor to not passing a ahrs_nav to ahrs_view ...